blueman/plugins/applet/DBusService.py

```diff
@@ -69,9 +78,11 @@ def reply(rfcomm):
             else:
                 if not self.parent.Plugins.run_ex("service_connect_handler", cb, service, ok, err) \
                         and isinstance(service, (SerialService, NetworkService)):
-                    service.connect(reply_handler=ok, error_handler=err)
+                    service.connect(reply_handler=lambda *args: ok(), error_handler=err)
```

Comment:
What is the point of ``*args``?

Reply:
`SerialService.connect` and `NetworkService.connect` expect the callback to take a `str` argument (see their already present signatures). We do not care here but we at least care about it for `SerialService` in `rfcomm_connect_handler` methods. Doesn't look like we actually use it anywhere for `NetworkService` (it contains the network interface name which we always grab with the `property-changed` signal of `AnyNetwork`), but I think it makes sense that it returns it, so I'd leave it as it is.